model/src/main/kotlin/org/digma/intellij/plugin/model/rest/event/LatestCodeObjectEventsRequest.kt

```diff
@@ -0,0 +1,3 @@
+package org.digma.intellij.plugin.model.rest.event
+
+data class LatestCodeObjectEventsRequest(val environments: List<String>,val fromDate: String)
```

Comment:
fromDate should be `DateTime`
```suggestion
data class LatestCodeObjectEventsRequest(val environments: List<String>,val fromDate: DateTime)
```